### PR TITLE
Ecse 1412 - just re-arrainging the schema, broke some CMR tests that were fixed.

### PR DESCRIPTION
**Pull request by eereiter**

### CODE REVIEW

**Review by TylerHeald1 — APPROVED** [review]

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj [hunk 1]
```diff
@@ -115,42 +115,49 @@
...
-                                             (assoc op :OrbitPeriodUnit "Decimal Minute")
-                                             op)
-                                           (if (:InclinationAngle op)
-                                             (assoc op :InclinationAngleUnit "Degree")
-                                             op)
-                                           (if (:StartCircularLatitude op)
-                                             (assoc op :StartCircularLatitudeUnit "Degree")
-                                             op)))}))
+  (let [[spatial] (select doc "/DIF/Spatial_Coverage")
+        [o] (select spatial "Orbit_Parameters")]
```
Comment (jceaser) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj:136]: I get yelled at when I do this in python by the linter, can you maybe give a longer name?

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/spatial.clj [hunk 2]
```diff
@@ -108,27 +114,25 @@
...
-                                                    :OrbitPeriodUnit "Decimal Minute")
-                                          op)
-                                        (if (:InclinationAngle op)
-                                          (assoc op :InclinationAngleUnit "Degree")
-                                          op)
-                                        (if (:StartCircularLatitude op)
-                                          (assoc op :StartCircularLatitudeUnit "Degree")
-                                          op)
-                                        (dissoc op :Period))}
+     :OrbitParameters              (when-let [[o] (select spatial "OrbitParameters")]
```
Comment (jceaser) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/spatial.clj:117]: same comment about o

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/shared_iso_parsing_util.clj [hunk 3]
```diff
@@ -58,21 +58,24 @@
...
-              (catch Exception e
-                (errors/throw-service-error
-                 :invalid-data (format "Error parsing the field %s with value %s" k v))))))))
+    (when (and (not (nil? map))
+               (not (empty? map)))
+      (if (or (nil? number-key-list)
+              (empty? number-key-list))
+        map
+      (into {}
+            (for [[k v] map]
```
Comment (jceaser) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/shared_iso_parsing_util.clj:67]: k v seams pretty standard across languages for this sort of thing, I do NOT apply my comment here (reluctantly)

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj [hunk 4]
```diff
@@ -333,40 +334,47 @@
...
+        geodetic-model (when-let [geo (parse-geodetic-model doc)]
+                         {:GeodeticModel geo})
+        local-coord-sys (when-let [coord (parse-local-coord-sys doc)]
+                          {:LocalCoordinateSystem coord})
+        hor-res (when-let [hr (parse-horizontal-data-resolutions doc)]
+                  {:HorizontalDataResolution hr})
+        resolution (when (or description geodetic-model local-coord-sys hor-res)
+                     {:ResolutionAndCoordinateSystem (umm-c/map->ResolutionAndCoordinateSystemType
+                                                      (merge {} description geodetic-model local-coord-sys hor-res))})
+        geometry (when-let [g (parse-geometry doc extent-info sanitize?)]
```
Comment (jceaser) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj:372]: more single letter variable names

**Review by jceaser — APPROVED** [review]
functionally this looks great, just suggest some minor changes to variable names if it is acceptable to the author.

**Review by zimzoom — APPROVED** [review]

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj (hunk 1 above)
Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj:136]: I changed the name to orbit-parameters

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/spatial.clj (hunk 2 above)
Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/spatial.clj:117]: same change - renamed to orbit-parameters

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj (hunk 4 above)
Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj:372]: I updated them.

**Pull request merged**